whole-tale/ngx-dashboard
# fix: handle wt_instance_deleted

## Problem
Fixes #148 

## Approach
* Handle `wt_instance_deleted` and `wt_instance_deleting` in Tale Catalog view
* Handle `wt_instance_deleted` in Run Tale view

## How to Test
Prerequisites: at least one Tale created and running (error state is ok)

1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. On the My Tales view (or any tab really), Stop a running Tale (or click on `Error!` if in error state)
    * You should see the Tale move out of the "Running Tales" panel and appear below
4. On the Run Tale > Interact view, Stop a running Tale
    * You should see the view update to reflect that the Tale is no longer running
